src/mame/layout/stepstag.lay

```diff
@@ -9,10 +9,10 @@ license:CC0
 			<bounds x="0" y="0" width="9" height="13.2" />
 		</screen>
 		<screen tag="screen">
-			<bounds x="9.1" y="0" width="16" height="13.2" />
+			<bounds x="9.1" y="0" width="19.36" height="13.2" />
```

Comment:
Where are you getting this number from?  For a 4:3 screen, the width should be 17.6 if the height is 13.2 – the width used doesn’t look like a common aspect ratio to me.

Reply:
This is pixel-aspect: 352*240.

Reply:
The default view should be using the physical aspect ratio of the monitors used, not an unnatural ratio to produce square pixels.

Reply:
To maintain many circles/balls used in this game look round(not oval), this hardware uses pixel-aspect ratio.
Once I observed a real Stepping Stage machine, and noticed two black borders alone the two long-sides of the CRTs(looks under-scan), just like when a PAL TV playing Nintendo NES or SEGA MegaDrive.